internal/kokoro/environment.sh

```diff
@@ -86,6 +86,9 @@ if [[ $ENVIRONMENT == *"kubernetes"* ]]; then
   mkdir -p ~/.local/bin
   mv ./kubectl ~/.local/bin
   export PATH=$PATH:~/.local/bin/
+  # install auth plugin
+  apt-get install google-cloud-sdk-gke-gcloud-auth-plugin
```

Comment:
This is fine for now, but likely this should be baked into our base image if needed instead of reinstalling on every rerun. Would you file an issue for this?